clarity_ext/units.py

```diff
@@ -0,0 +1,30 @@
+import math
+import logging
+
```

Comment:
I think small classes like this should find it's home in some common module, like "misc".